options/locale/locale_en-US.ini

```diff
@@ -1053,6 +1053,13 @@ form.reach_limit_of_creation_n = The owner has already reached the limit of %d r
 form.name_reserved = The repository name "%s" is reserved.
 form.name_pattern_not_allowed = The pattern "%s" is not allowed in a repository name.
 
+sidebar.readme = Readme
+sidebar.stars = stars
+sidebar.forks = forks
+sidebar.watching = watching
+sidebar.contributors = Contributors
+latest_release = Latest
```

Comment:
Move all these translation to top-level imho, e.g. translation key `readme`, `stars` etc.

Reply:
Done.